bootstrap/src/lib/cache/index.ts

```diff
@@ -37,16 +38,20 @@ const defaultOptions = () => ({
     entropyMax: Number(env.REQUEST_COALESCING_ENTROPY_MAX) || DEFAULT_RC_ENTROPY_MAX,
   },
 })
+export type CacheOptions = ReturnType<typeof defaultOptions>
+
 const defaultCacheOptions = () => {
   const type = env.CACHE_TYPE || DEFAULT_CACHE_TYPE
   const options = type === 'redis' ? redis.defaultOptions() : local.defaultOptions()
   return { ...options, type }
 }
+export type ImplCacheOptions = ReturnType<typeof defaultCacheOptions>
```

Comment:
Maybe switch the words:
```suggestion
export type CacheImplOptions = ReturnType<typeof defaultCacheOptions>
```